lib/db/query.js

```diff
@@ -129,7 +130,68 @@ async function getSideload() {
   };
 }
 
+async function getTestBody(testId) {
+  const query1 = `
+  SELECT 
+    t.name,
+    t.run_frequency_mins as frequency,
+    t.headers,
+    t.payload as body,
+    t.url,
+    hp.name as method
+  FROM tests t
+    JOIN http_methods hp ON hp.id = t.method_id 
+    WHERE t.id = ${testId};
+  `;
+
+  const query2 = `
+  SELECT 
+   a.type,
+   a.property,
+   a.expected_value,
+   ct.name
+  FROM assertions a
+    JOIN comparison_types ct
+    ON a.comparison_type_id = ct.id
+    WHERE a.test_id = ${testId};
+   `;
+
+  const query3 = `
+    SELECT 
+      DISTINCT r.name
+    FROM regions r
+      JOIN test_runs tr ON r.id = tr.region_id
+      WHERE tr.test_id = ${testId};
+    `;
+
+  const testCofnig = await dbQuery(query1);
+  const assertions = await dbQuery(query2);
+  const locations = await dbQuery(query3);
+
+  // eslint-disable-next-line object-curly-newline
+  const { name, frequency, headers, body, url, method } = testCofnig.rows[0];
+
+  const json = {
+    test: {
+      title: name,
+      locations: locations.rows.map((obj) => helpers.toDash(obj.name)),
+      minutesBetweenRuns: frequency,
+      type: 'API',
+      httpRequest: {
+        method,
```

Comment:
I believe `testCongif.method` is lower case in it's current form. CRUD expects method to be all upper case

Reply:
Looks good!